src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/python/organizations.py

```diff
@@ -22,9 +22,14 @@ class Organizations: # pylint: disable=R0904
     _config = Config(retries=dict(max_attempts=30))
 
     def __init__(self, role, account_id=None):
+        self.role = role
         self.client = role.client(
             'organizations',
             config=Organizations._config)
+        self.tags_client = role.client(
+            'resourcegroupstaggingapi',
+            region_name='us-east-1',
+            config=Organizations._config)
```

Comment:
Would be great to build this client from inside the Organizations class, so we don't need to access private variables like `_config`.
Alternatively we can change the config to be public as well, but I would prefer adding it to the class instead.

Reply:
This is done exactly like the other client uses in the same class.

Reply:
Alright, I think we should get those changed too in that case. But having that in a separate PR would make sense I guess.